### ISSUE TITLE
Big performance degrade

### ISSUE THREAD

**Issue by mawis** [luoliyan/chinese-support-redux#48]
I noticed a big performance degrade after updating to version 0.10.0 of the plugin.

**What I do**
1. I have opened a card in the card overview (keyboard shortcut B).
2. Cursor blinks in the first field (汉字 in my case).
3. I press the TAB key on my keyboard.

**What happens**
1. For several seconds nothing happens at all.
2. After several seconds the cursor switches to the second input field (繁體字 in my case).

**What I would expect**
Cursor switches to the 繁體字 field without a noticable delay.

**Specs:**
 - OS: Windows
 - Anki Version 2.1.4
- Chinese Support Version 0.10.0

**Additional context**
- I have around 3500 notes (more than 10,000 cards)
- The note template has been extended to 11 fields (汉字, 繁體字, 量詞,  意思, 拼音, 彩色, ㄅㄆㄇㄈ, RubyBPMF, Ruby, Alternative, Sound)

**Comment by luoliyan** [comment]
So just to clarify, these are existing cards with the audio already downloaded?

**Comment by mawis** [comment]
Yes, audio is already present in the note. Also I disabled audio download completely in the plugin as I download audio files manually from Forvo.
